lightwood/api/json_ai.py

```diff
@@ -298,7 +301,9 @@ def add_implicit_values(json_ai: JsonAI) -> JsonAI:
             models[i]['args']['timeseries_settings'] = models[i]['args'].get(
                 'timeseries_settings', '$problem_definition.timeseries_settings')
             models[i]['args']['net'] = models[i]['args'].get(
-                'net', '"DefaultNet"' if not problem_definition.timeseries_settings.is_timeseries else '"ArNet"')
+                'net', '"DefaultNet"' if not problem_definition.timeseries_settings.is_timeseries
+                or not problem_definition.timeseries_settings.use_previous_target
```

Comment:
So wait, if we "use previous target" than we want to use the `DefaultNet` not the `ArNet` ... why ?

Reply:
Other way around: if we _don't_ "use previous target" then we cannot use ArNet, as it uses the `TsPlainEncoder` that passes raw previous values to the model.